DB/academicmanager.sql

```diff
@@ -124,3 +117,24 @@ ALTER TABLE `course`
 --
 ALTER TABLE `person`
   MODIFY `personId` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=13;
+
+
+-- DROP the columns below as requested by G.Metaxas
+ALTER TABLE person DROP IF EXISTS username;
+ALTER TABLE person DROP IF EXISTS password;
+
+
+-- Creation of table users
+
+CREATE TABLE IF NOT EXISTS users (
+id int NOT NULL,
+user_email varchar (255) NOT NULL,
+user_password char (128),
+last_login TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+date_created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+status_account boolean,
+personId int (11),
+PRIMARY KEY(user_email),
```

Comment:
The primary key should be id, not user_email. User_email may be an indexed column/unique constraint, etc